uiowa/uiowa
# Ctools 4

Resolves #7395 

I thought I was being clever before when I made sure that our views block display allow setting had the same key name (`configure_filters`) as the feature in Ctools Views. However, this turned out to be a mistake because it meant it was impossible to leave our filters turned on and keep the new filters turned off. To get around this, I changed the key name for our filters to `configure_filters_custom`.  This allowed us to keep the new feature in the module turned off and our existing custom feature turned on.

As such, I ended up going through and re-saving all the views that contained this key to make sure I was picking up all the relevant changes. Apparently, if you don't click into the allow settings and you save a view, it doesn't include them in the display's settings in config. I specifically clicked the allow settings in every views block display to make sure they were up-to-date and consistent. This means that some of the views config seemingly contains new `allow` settings for some block displays. Examining these should reveal that everything is "turned off".

# How to test

- Review [file changes](https://github.com/uiowa/uiowa/pull/7396/files).
- Test article and people list blocks to ensure no functionality has changed.
- Test as many of the custom list blocks as is necessary to convince yourself that nothing has changed.
